src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.html

```diff
@@ -22,3 +22,27 @@
                          [selection]="expandedRow">
   </cd-rgw-bucket-details>
 </cd-table>
+
+<ng-template #bucketSizeTpl
+             let-row="row">
+  <cd-usage-bar *ngIf="(row.bucket_quota.max_size > 0); else noSizeQuota"
+             [totalBytes]="1"
+             [usedBytes]="row.size_usage">
+  </cd-usage-bar>
+
+  <ng-template #noSizeQuota>
+  {{row.size_usage | dimlessBinary}}
```

Comment:
As discussed further down in this file.
How about a more directed approach that doesn't mix the variable number scope?
```suggestion
  {{row.bucket_size | dimlessBinary}}
```

Reply:
Makes sense, will apply the changes